src/_angular/font-setting/dtv-font-setting.js

```diff
@@ -383,6 +410,51 @@
               sheet.addRule("@font-face", rule);
             }
           }
+
+          function addVerticalAlignButtons(editor) {
+            editor.addButton("aligntop", {
+              image: "//s3.amazonaws.com/Rise-Images/Icons/align-top.svg",
```

Comment:
@stulees True. I was just thinking that our repos are already doing that for some files anyway (e.g. copying FontAwesome file and even some images). I guess it's a question of whether we want to be consistent with how we've been doing things thus far, or if this way is better (not sure?), then start doing things this way instead.